braze/client.py

```diff
@@ -400,3 +400,36 @@ def send_canvas_message(
         }
 
         return self._post_request(message, BrazeAPIEndpoints.SEND_CANVAS)
+
+    def change_email_subscription_state(
+        self,
+        email,
+        subscription_state,
```

Comment:
A change suggestion: Reading the ticket (title reads: "Write script to automate braze unsubscription") it seems that the primary use case for this is it unsubscribe users. There are a number of US and international laws that have penalties attached to them should someone unsubscribe and continue to receive emails. 

I would like to see us limit this function to unsubscribe only. If we do not have a use case to change this state to anything other than unsubscribe, this method should only do that. This will prevent us from accidentally resubscribing people we did not intend to and prevent us from shooting ourselves in the foot and potentially causing 2u/edX to be fined.

Reply:
PR updated with this suggested change.